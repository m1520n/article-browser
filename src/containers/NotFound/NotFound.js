import React from 'react';

const NotFound = () => <div>Ooops! The page you requested was not found...</div>;

export default NotFound;
