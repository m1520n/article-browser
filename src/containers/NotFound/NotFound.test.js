import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './NotFound';

test('NotFound is rendered correctly', () => {
  const component = renderer.create(<NotFound />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
