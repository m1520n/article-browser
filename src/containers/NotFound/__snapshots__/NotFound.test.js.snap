// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound is rendered correctly 1`] = `
<div>
  Ooops! The page you requested was not found...
</div>
`;
